index.html

```diff
@@ -343,8 +343,11 @@
           <li>If the <a>browsing context</a> of the script calling the
           constructor is a <a>nested browsing context</a> whose origin is
           different from the <a>top-level browsing context</a>'s origin and the
-          nested browsing context is not <a>allowed to make payment
-          requests</a>, then <a>throw</a> a <a>SecurityError</a>.
+          nested <a>browsing context</a>'s <a>browsing context container</a> is
+          an <a><code>iframe</code></a> element whose <a>node document</a> is
```

Comment:
> Remove `iframe` check here I think.

OK, removed in https://github.com/w3c/browser-payment-api/pull/359/commits/681837dea0c5232d2ef3c97fa87dfc447fbac479